.gitignore

```diff
@@ -0,0 +1,6 @@
+.DS_Store
```

Comment:
Talking about project files… What do you think about an [`.editorconfig`](http://editorconfig.org/)? It helps reduce all the fight on code style. Also, it is again a matter of making it easy for people to contribute.